enhancements/monitoring/monitoring-stack-operator.md

```diff
@@ -0,0 +1,490 @@
+
+## Summary
+
+This document proposes a solution and a delivery plan for scraping, querying and alerting on metrics related to Red Hat Managed Services running in OSD clusters. The proposed solution is not intended to complement the existing monitoring stack managed by the Cluster Monitoring Operator, nor used directly by Customers at this moment.
+
+### Glossary
+
+* MS-SRE/MT-SRE: Managed Service / Managed Tenant Site Reliability Engineer
+* MTO: Managed Tenant Owners - the teams that develop each Managed Service. The Kafka Team is one example of an MTO.
+* RHOBS: Red Hat Observability Service
+* RHODF: Red Hat OpenShift Data Foundation
+
+For additional acronyms, see [Red Hat Dictionary](https://source.redhat.com/groups/public/red-hat-dictionary-or-lexicon/red_hat_abbreviations_acronyms_and_initialisms_dictionary_wiki)
+
+## Background
+
+OpenShift Monitoring is currently composed of two main systems - Platform Monitoring (PM) and User Workload Monitoring (UWM), both of which are part of OpenShift core.
+
+Platform Monitoring holds metrics about core system components running in `openshift-*` and `kube-*` namespaces. Platform Monitoring is enabled in all OpenShift installations and cannot be disabled. Some of the components scraped by PM include core-dns, etcd, kube-apiserver, image-registry, operator-lifecycle-manager etc.
+Platform Monitoring is by design an immutable stack, [which means that admins cannot extend it with additional metrics without losing the support.](https://docs.openshift.com/container-platform/4.7/monitoring/configuring-the-monitoring-stack.html#support-considerations_configuring-the-monitoring-stack)
+
+User Workload Monitoring is an optional stack managed by Cluster Monitoring Operator (CMO) and is disabled by default. Once enabled by cluster admins, it allows users to monitor namespaces that are not part of OpenShift core.
+Please refer to the User Workload Monitoring [enhancement proposal](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/user-workload-monitoring.md) for more details. If enabled, only one stack can be deployed. The recommended way to get “platform” data is through Thanos Querier. Alternatively, Platform Monitoring Prometheus can also federate the latest data.
+
+Recently an essential model of managing software for customers emerged at Red Hat, referred to as “Managed Services” or “Managed Tenants”. From an architectural point there are three types of such services:
+
+* Addons/Layered Services
+* SaaS
+* A combination of the first two (SaaS that requires an Addon)
+
+The way these managed services are intended to run is presented in the following diagram.
+
+![drawing](https://docs.google.com/drawings/d/e/2PACX-1vQrMBfwE8DJm3kVtC6SLVYJLEztJRABIYwvILMgtOmyyE6WqKSCqvZHksUsVUoqSZIt5E_4ro4ObGmY/pub?w=1312&h=511)
```

Comment:
Please copy external references like diagrams into the repo with the design document, so they can be versioned and published without requiring access to Google Apps. Since there are quite a few figures for this enhancement, I suggest creating a folder with the same name as the base document and placing all of the images inside that folder.